dotnet/src/SemanticKernel.Abstractions/Contents/ChatMessageContent.cs

```diff
@@ -15,6 +15,12 @@ namespace Microsoft.SemanticKernel;
 /// </summary>
 public class ChatMessageContent : KernelContent
 {
+    /// <summary>
+    /// Name of the author of the message
+    /// </summary>
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? Name { get; set; }
```

Comment:
Changes to the public API should not be immediately GA. We should mark the changes as experimental to allow us to get some feedback.

Reply:
SKEXP0011